Source/DafnyLanguageServer/Language/Symbols/Symbol.cs

```diff
@@ -6,6 +6,14 @@ namespace Microsoft.Dafny.LanguageServer.Language.Symbols {
   /// Represents a resolved symbol.
   /// </summary>
   public abstract class Symbol : ISymbol {
+
+    /// <summary>
+    /// Helper to convert a potentially null symbol to a list of symbols
+    /// </summary>
+    public static IEnumerable<TSymbol> AsEnumerable<TSymbol>(TSymbol? symbol) where TSymbol : ISymbol {
```

Comment:
I think it makes sense to extract this method into a distinct class, for example `SymbolExtensions`. Then you can declare it as an extension method making the usage more fluent, for example:
```cs
methodSymbol.AsEnumerable()
```

This extension method may look like this, avoiding the allocation of a new array:
```cs
public static IEnumerable<TSymbol> AsEnumerable<TSymbol>(this TSymbol? symbol) where TSymbol : ISymbol {
  if(symbol != null) {
    yield return symbol;
  }
}
```

Reply:
Clever. I found the existing class of SymbolExtensions and added it there.